packages/switch/NativeControl.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class NativeControl extends React.Component {
+  render() {
+    const {
+      rippleActivatorRef,
+      ...otherProps
+    } = this.props;
+
+    return (
+      <input
+        type='checkbox'
+        role='switch'
+        className='mdc-switch__native-control'
```

Comment:
Do you think people will use this component on its own? If so we should merge this.props.className and 'mdc-switch__native-control'